samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -276,9 +276,10 @@ static void cmd_send(struct k_work *work)
 	}
 
 done:
+	k_sleep(500); /* allow time for TX DMA */
```

Comment:
it shouldn't be needed. It's workaround that hides real problem.

Reply:
@nordic-krch as discussed by email, I keep it for now since it's effective